apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/CloudTrace.cs

```diff
@@ -80,6 +80,25 @@ public sealed class CloudTrace
         public static IManagedTracer CurrentTracer =>
             TracerManager.GetCurrentTracer() ?? DoNothingTracer.Instance;
 
+        /// <summary>
+        /// Creates a <see cref="TraceHeaderPropagatingHandler"/> to propagate trace headers
+        /// in Http requests.
+        /// <example>
+        /// <code>
+        /// public void DoSomething()
+        /// {
+        ///     var traceHeaderHandler = CloudTrace.CreateHandler();
+        ///     using (var httpClient = new HttpClient(traceHeaderHandler))
+        ///     {
+        ///         ...
+        ///     }
+        /// }
+        /// </code>
+        /// </example>
+        /// </summary>
+        public static TraceHeaderPropagatingHandler CreateHandler() =>
```

Comment:
I think we may want to rename this to be more specific - CreateTracingHttpMessageHandler is possibly overkill, but possibly CreateHttpMessageHandler or CreateHttpHandler? Something that screams "This is what you want if you're doing HTTP work, but otherwise it's not."